### DISCUSSION TITLE
TalkElixir中文社区未来演进想法征集

### DISCUSSION THREAD

**Discussion by gofenix** [TalkElixir/night#19] [Ideas]
新的一年，TalkElixir整装待发，欢迎Elixir小伙伴们来一起学习分享！

大家有什么好的想法或者建议，可以在这里回复，大家一起把Elixir中文社区做大做强！

**Comment by zhenyuanlau** [comment]
个人感觉依托开源项目来开展可能会更有凝聚力一些，可以借鉴 go 中文社区建设。

对比图科技分享的内容进行二次挖掘和梳理也能有不少收获，真心希望比图科技能在国内社区建设上有所建树。
